apps/client/src/widgets/close_zen_button.ts

```diff
@@ -28,6 +29,10 @@ const TPL = /*html*/`\
         -webkit-app-region: no-drag;
     }
 
+    body.zen .mobile-close-zen-container {
+        top: -2px;
+    }
```

Comment:
No need for a dedicated class name for mobile, you could do `body.mobile.zen`

```suggestion
    body.zen.mobile .close-zen-container {
        top: -2px;
    }
```